recipes/libqasm/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake, cmake_layout
+from conan.tools.files import get
+from conan.tools.scm import Version
+import os
+
+class LibqasmConan(ConanFile):
+    name = "libqasm"
+
+    # Optional metadata
+    license = "Apache-2.0"
+    homepage = "https://github.com/QuTech-Delft/libqasm"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Library to parse cQASM files"
+    topics = ("code generation", "parser", "compiler", "quantum compilation", "quantum simulation")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_python": [True, False],
+        "build_tests": [True, False],
```

Comment:
I don't think that test like option can be accepted

Reply:
Hi @SpaceIm, many thanks for your review!

I think I had previously spoken with you through Slack, and you had commented to me the idea of building the tests always by default and then using something like `skip_tests` to avoid that build. I have two comments regarding that:

1. I don't like the idea of building tests by default. I would like the uninformed user that is building my package to get a quick and clean, without tests and test binaries, build.

2. In case the `skip_tests` had other benefits that I cannot yet see, how could I connect it with the `CMakeLists.txt`? I understand it would be with a line such as `tc.variables["LIBQASM_BUILD_TESTS"] = not self.options.skipt_tests`?

Thanks!

Reply:
Using `self._should_build_test` now.